prometheus-client/src/Prometheus/Metric/Histogram.hs

```diff
@@ -74,6 +78,20 @@ instance Observer Histogram where
     -- | Add a new observation to a histogram metric.
     observe h v = withHistogram h (insert v)
 
+-- | Observe that a particular floating point value has occurred.
+-- For example, observe that this request took 0.23s.
+--
+-- In addition, tracks a set of key-value pairs called an [exemplar](https://grafana.com/docs/grafana/latest/fundamentals/exemplars/)
+-- which are used to correlate time series data with traces.
+--
+-- Typically the values tracked are "trace_id" and "span_id".
+--
+-- This feature is experimental and must be [enabled on the Prometheus server](https://prometheus.io/docs/prometheus/latest/feature_flags/).
+--
+-- > withLabel incomingHttpRequestSeconds "Signup_POST" (`observeWithExemplar` 1.23 [("trace_id", "12345"), ("span_id", "67890")])
```

Comment:
In a previous commit I had a tiny helper called `traceLabel` and `traceAndSpanLabels` that would construct this. I could add that back if you'd like

Reply:
Noting to myself there's a syntax error here: (`observeWithExemplar` 1.23 [("trace_id", "12345"), ("span_id", "67890")])